cfgov/v1/preprocessed/css/molecules/text-introduction.less

```diff
@@ -0,0 +1,60 @@
+/* topdoc
+  name: Theme variables
+  family: cfgov-molecules
+  notes:
+    - "The following color and sizing variables are exposed,
+       allowing you to easily override them before compiling."
+  patterns:
+  - name: Colors
+    codenotes:
+      - |
+        @m-text-introduction_rule
+  tags:
+  - less
+*/
+@m-text-introduction_rule: @gray-50;
```

Comment:
Shouldn't this rule be the same as all our other rules. Hopefully, we'd never want to change some rules and not others.